### DISCUSSION TITLE
Can it work with dangerouslySetInnerHTML?

### DISCUSSION THREAD

**Discussion by DrazhinUstin** [anantoghosh/react-linkify-it#49] [Q&A]
I have html string and I am trying to do this:
```
 <LinkItEmail className='text-primary hover:underline'>
      <div dangerouslySetInnerHTML={{ __html: content }} />
  </LinkItEmail>
```
But it doesn't work. Can I somehow solve this problem?

**Comment by anantoghosh** [comment]
Would have to reimplement some things:
This is how I could get it to work.
https://codesandbox.io/p/sandbox/react-linkify-it-forked-m5xhs4?workspaceId=4c157d8e-4f26-4564-9461-dc8211f82273

You can copy urlRegex too then you would not need this library.
